app/assets/stylesheets/application.css

```diff
@@ -1,15 +1,78 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css");
+
+.layout_body {
+  background-color: #f1f3f2;
+}
+
+.home_body {
+  margin-top: 20%;
+}
+
+.logo {
+  max-height: 50%;
+  text-align: center;
+  font-size: xx-large;
+  font-weight: bolder;
+}
+
+.account {
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.acc_bttn {
+  margin: 2%;
+  width: 100%;
+}
+
+.nav_bar {
+  background-color: #3778c2 !important;
+}
+
+.white_text {
+  color: white;
+}
+
+.header {
+  background-color: #3778c2;
+  color: white;
+  padding: 20px 20px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.field {
+  width: 95%;
+  margin: 2% auto;
+}
+
+.back_btn {
+  margin-left: 50%;
+  font-size: 25px;
+  color: white;
+}
+
+.action_btn {
+  margin-right: 5%;
+}
+
+.forget_pw_btn :active {
+  color: grey;
+}
+
+.home_icons {
+  width: 500px;
+}
+
+.btn_new_cat {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
```

Comment:
- [ ] `optional`
- [ ] For accessibility and making a good `UI` i suggest that button a bit up there so if can be visible 

![image](https://user-images.githubusercontent.com/66230306/219868413-0fa4e678-8178-4199-81bf-fe5311392256.png)